lib/github_api/client/orgs/projects.rb

```diff
@@ -22,9 +22,11 @@ class Client::Orgs::Projects < API
     #
     # @api public
     def list(*args)
-      arguments(args, required: [:org_name])
-
+      arguments(args, required: [:org_name]) do
+        permit %w[ state ]
```

Comment:
I wouldn't use `permit` any more. It is present in the old code but it proved to be poor choice for the maintenance of the library. GitHub quite often changes what parameters can be passed, whether it is the name or the actual number of parameters and I feel it is generally more flexible and safer to allow for any options.